exercises/ex_atm.py

```diff
@@ -0,0 +1,137 @@
+# ATM Exercise
+
+import sys
+
+FILE_PATH = 1
```

Comment:
good use of constants.

However is this case the name is a but unclear. FILE_PATH sounds like the string telling me where the file goes not the index of the file path in the arguments